scripts-dev/old_deps.py

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env python
+
```

Comment:
```suggestion
#
```

... just for consistency with everywhere else.